### PR TITLE
S4 center dialog

### PR DESCRIPTION
**Pull request by niqt**
#### :thinking: What?

Dialog in the center of the screen for application that doesn't use 100vh

#### :man_shrugging: Why?

Required from Squad 4

#### :pushpin: Jira Issue

 Not tracked.

#### :no_good: Blocked by

Nothing.

### CODE REVIEW

File: src/UI/Internal/Dialog.elm [hunk 1]
```diff
@@ -68,6 +67,12 @@ desktopDialogView cfg (Dialog.Dialog { title, icon } { body, buttons }) =
         [ body
         , buttonsRow cfg buttons
         ]
+        |> Element.el
+            [ Element.centerX
+            , htmlAttribute (Html.Attributes.style "position" "fixed")
+            , htmlAttribute (Html.Attributes.style "top" "50%")
+            , htmlAttribute (Html.Attributes.style "bottom" "50%")
```
Comment (PedroHLC) [inline src/UI/Internal/Dialog.elm:74]: Does it not work when put in the `Element.column` attributes?

Comment (niqt) [reply src/UI/Internal/Dialog.elm:74]: No, has different behaviour

**Review by PedroHLC — APPROVED** [review]
This one looks good for me, easier to understand what's going on.

**Comment by PedroHLC** [comment]
Like on #333, you'll need to bump the version...

**Comment by PedroHLC** [comment]
Also, this broke the storybook...

**Pull request closed without merging**